### PR TITLE
[mltest] Set batchSize where it makes sense

### PR DESCRIPTION
**Pull request by bertmaher**
If I understand correctly, we should always set EE's batchSize equal to the batch dimension of the training tensors, so that SGD is done "properly".  (If I don't understand correctly, this diff is a good opportunity to correct me :-D ).  I guess this doesn't really "matter", but it makes the test code easier to understand for folks like me who are learning ML while trying to write a test case.

### CODE REVIEW

**Comment by nadavrot** [comment]
Your change LGTM. For other people reviewing this patch, the batchSize is multiplied with the learning rate in SGD, so it's more a matter of convention then correctness.
